apps/back-end/src/dataset.ts

```diff
@@ -162,8 +164,8 @@ export class DatasetWriter {
       // they stand out as different. The result is intentionally somewhat CSV-like.
       searchableFile.write(
         `{   "itemProps":\n` +
-          JSON.stringify(filterablePropNames) +
```

Comment:
Whitespace change - not a problem in itself but it looks like your editor is not applying the PrettierJS formatting styles used in the project. We agreed to use this back at the start of the project to eliminate "stylistic churn" - exactly this formatting change which depends on the idiosyncrasies of an individual's editor config or personal style.

Suggest addition of the changes from "npm run prettier:format" at the end of the PR to work around that, and addition of the PrettierJS plug-in for VSCode. I think you can turn it on in a per-project way but I'm not an expert, @rogup might know better.